app/assets/javascripts/discourse/app/controllers/user-private-messages-group.js

```diff
@@ -0,0 +1,41 @@
+import I18n from "I18n";
+import Controller, { inject as controller } from "@ember/controller";
+import { computed } from "@ember/object";
+
+export default class extends Controller {
+  @controller user;
+
+  get viewingSelf() {
+    return this.user.viewingSelf;
+  }
+
+  @computed(
+    "pmTopicTrackingState.newIncoming.[]",
+    "pmTopicTrackingState.statesModificationCounter"
+  )
+  get newLinkText() {
+    return this.#linkText("new");
+  }
+
+  @computed(
+    "pmTopicTrackingState.newIncoming.[]",
+    "pmTopicTrackingState.statesModificationCounter"
+  )
+  get unreadLinkText() {
+    return this.#linkText("unread");
+  }
+
+  #linkText(type) {
+    const count =
+      this.pmTopicTrackingState?.lookupCount(type, {
+        inboxFilter: "group",
+        groupName: this.groupName,
+      }) || 0;
```

Comment:
Based in that implementation, I don't think we need a fallback to 0 

https://github.com/discourse/discourse/blob/main/app/assets/javascripts/discourse/app/services/pm-topic-tracking-state.js#L77